search/src/main/resources/application.yml

```
@@ -98,8 +100,8 @@ resilience4j:
         waitDurationInOpenState: 20s
 
 
-logging:
-  config: classpath:logback-spring.xml
-  level:
-    org.hibernate.SQL: DEBUG
-    org.hibernate.type.descriptor.sql.BasicBinder: TRACE
\ No newline at end of file
+#logging:
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)